docs/api/covidcast_changelog.md

```diff
@@ -46,7 +46,7 @@ Duplicate survey weights had corrupted historical figures for the following sign
   
 #### 20 November 2020  
 
-Due to a bug in our data processing system, estimates of the percentage of people reporting that they have been tested for COVID-19 calculated before October 8th were incorrect. 
+Due to a bug in our data processing system, estimates of the percentage of people reporting that they have been tested for COVID-19 calculated before October 8th was incorrect. 
```

Comment:
the verb here goes with estimates, not percentage. Open to a less contorted wording though, that's a doozy.

```suggestion
Due to a bug in our data processing system, estimates of the percentage of people reporting that they have been tested for COVID-19 calculated before October 8th were incorrect. 
```